scripts/working-with-js-objects-and-values.t

```diff
@@ -71,16 +68,19 @@ file. To update the tests, run `dune build @extract-code-blocks`.
   $ dune build @melange
 
   $ cat > input.ml <<\EOF
+  > 
+  > external map : 'a array -> 'b array -> (('a -> 'b -> 'c)[@u]) -> 'c array = "map"
+  > 
   > let add x y = x + y
   > let _ = map [||] [||] add
   > EOF
 
   $ dune build @melange
-  File "input.ml", line 2, characters 8-11:
-  2 | let _ = map [||] [||] add
-              ^^^
-  Error: Unbound value map
-  Hint: Did you mean max?
+  File "input.ml", line 5, characters 22-25:
+  5 | let _ = map [||] [||] add
+                            ^^^
+  Error: This expression has type int -> int -> int
+         but an expression was expected of type ('a -> 'b -> 'c [@u])
```

Comment:
Is this expected?

Reply:
Yes, it's expected. This example is supposed to not compile because `map` expects an uncurried function as its third argument, but `add` is curried.

It's sometimes hard to figure out which snippet is the one that's supposed to break because they aren't extracted in the order that they appear in the markdown file.

Reply:
I added comments in the prelude of snippets that aren't expected to compile in https://github.com/melange-re/melange-re.github.io/pull/211/commits/f79420b4be9d8e32731235068d9bffa03bdf84e4